sktime/annotation/tests/test_all_annotators.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
can we add more tests on the output, i.e. if format=dense, it should have the same index as X

Reply:
hm, whould we move the `format` arguments into the base class and pass them via `super`? That way it would be an expected interfacve point there, and can be read in the base class for checks.

Reply:
Yes, moving them to `super` sounds good to me